modules/nodes-cluster-admission-about.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-admission.adoc
+
+[id="nodes-containers-controllers-about_{context}"]
+= Understanding custom admission controllers
+
+Admission control plug-ins intercept requests to the master API prior to persistence of a resource, but after the request is authenticated and authorized.
+
+Each admission control plug-in is run in sequence before a request is accepted into the cluster. If any plug-in in the sequence rejects the request, the entire request is rejected immediately, and an error is returned to the end-user.
+
+Admission control plug-ins may modify the incoming object in some cases to apply system configured defaults. In addition, admission control plug-ins may modify related resources as part of request processing to do things such as incrementing quota usage.
+
+The {product-title} master has a default list of plug-ins that are enabled by
+default for each type of resource (Kubernetes and {product-title}): 
+
+* NamespaceLifecycle
+* LimitRanger 
+* ServiceAccount
+* PersistentVolumeClaimResize
+* DefaultStorageClass
+* DefaultTolerationSeconds
+* MutatingAdmissionWebhook
+* ValidatingAdmissionWebhook
+* ResourceQuota
+* Priority
```

Comment:
@ravisantoshgudimetla This list is from Kubernetes 1.13 docs. Is it accurate for OpenShift 4.1? Do the default admission controllers appear anywhere in code? The `kubeapiserver` object doesn't seem to list them.

Reply:
>  Do the default admission controllers appear anywhere in code?

AFAIK, you can find them when kube-apiserver starts.

Perhaps @openshift/sig-master can help.